pkgs/by-name/kd/kdotool/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  dbus
+}:
+
+let
+  pname = "kdotool";
```

Comment:
Can we move this into the package definition? I'm not sure if the Rust builders do `finalAttrs` but preferably that.